tests/RoyalmailTest.php

```diff
@@ -185,4 +424,67 @@ public function testRoyalMailClassNull()
         $this->assertEquals($this->emptyArray, $this->calculateMethodClass->getMethods(null, null, null));
     }
 
+    /**
+     * Test co compare the meta names vs the clean name to check that the correct value
+     * exists and is being used.
+     */
+    public function testMethodToMetaVsCleanName()
+    {
+        foreach ($this->dataClass->mappingMethodToMeta as $array => $data){
+            foreach ($this->dataClass->mappingCleanNameToMethod as $method => $methodData)
+            {
+                if ($data[0] ==  $methodData[2]){
+                    $this->assertEquals($data[4], $methodData[1]);
+                }
+            }
+        }
+    }
+
+    /**
+     * Test for insurance value checking that the correct insurance value is being
+     * used in the CSV files
+     */
+    public function testInsuranceValue()
+    {
+        foreach ($this->dataClass->mappingMethodToMeta as $array => $data){
+            foreach ($this->dataClass->mappingDeliveryToPrice as $method => $methodData)
+            {
+                if ($data[0] == $methodData[0]){
```

Comment:
These magic numbers probably should be moved out into class constants somewhere i.e.

const CSV_CLEAN_NAME_COLUMN = 0;